ink/yuki/punishment_edging

```diff
@@ -451,13 +476,13 @@ TODO set perform
 /perform
   I am a kind and generous mistress, and even though this is still a punishment, I want to see my good puppy make the most of it. 
   
-  I can see you need it. 
+  I can see you want the pleasure. 
```

Comment:
Maybe break up the line starting with "I am a kind and generous Mistress..." so that it will show better in subtitles in game.

For the second line, it's too formal and Yuki is more casual and enthusiastic. Perhaps try, "I know how much you really to."

Reply:
Broken up that long run-on line now, and I decided to actually just remove that other line all together, it feels better to me that way